provisioning/freeipa-users/defaults/main.yml

```diff
@@ -1,4 +1,10 @@
-freeipa_user:
+freeipa_users:
   - { user: "lisa", first: "lisa", last: "doe" }
   - { user: "linda", first: "linda", last: "doe" }
   - { user: "thomsen", first: "thomsen", last: "doe" }
+  - { user: "ldapuser1", first: "ldap", last: "user1" }
+  - { user: "ldapuser2", first: "ldap", last: "user2" }
+  - { user: "ldapuser3", first: "ldap", last: "user3" }
+  - { user: "ldapuser4", first: "ldap", last: "user4" }
+  - { user: "ldapuser5", first: "ldap", last: "user5" }
```

Comment:
These users are also provisioned on Sander's demo lab environment and used in the exercises throughout the book e.g. on p203 `ldapuser1` and p1113 `ldapuser5`